nctl-tests/scenarios/English_2_bid.sh

```diff
@@ -46,7 +46,7 @@ nctl-view-chain-deploy deploy=$BID_2_DEPLOY
 
 cd $CWD
 
-STATE=$(casper-client get-state-root-hash\
-  --node-address $NODE_1_ADDRESS\
-  | jq .result.state_root_hash\
-  | tr -d '"')
\ No newline at end of file
+# STATE=$(casper-client get-state-root-hash\
```

Comment:
This should be left uncommented in "root node" (setup, scenario) files, to make it easier to immediately query the chain interactively, which is something one would very likely want to do if using this

Reply:
Ok it was not obvious because as far as I can remember those variables STATE are not all printed to screen so it seems that succeeding or failing it would not warn the user. I can add a commented echo STATE after that or revert those changes no worries.

Reply:
No need to echo it, but having it set correctly is a time saver